lldb/include/lldb/API/SBProcess.h

```diff
@@ -398,6 +398,13 @@ class LLDB_API SBProcess {
   /// valid.
   lldb::SBProcessInfo GetProcessInfo();
 
+  /// Return target dump file during postmortem debugging.
+  /// An empty file will be returned for live debugging.
+  ///
+  /// \return
+  ///     The target dump file spec.
```

Comment:
"The path to the core file for this target or an invalid file spec if the process isn't loaded from a core file.